.github/workflows/sdks.yml

```diff
@@ -480,3 +480,115 @@ jobs:
           api-level: 29
           arch: x86_64
           script: ~/test-toolchain.sh
+  assemble-sdks:
+    runs-on: ubuntu-22.04
+    needs: [build-sdk-and-tests, get-latest-toolchain]
+    strategy:
+      fail-fast: false
+      matrix:
+        version: [release, devel, trunk]
+    steps:
+      - name: Extract the latest version string
+        id: version
+        run: |
+          if [[ ${{ matrix.version }} = 'release' ]]; then
+            TAG="${{ needs.get-latest-toolchain.outputs.release-version }}"
+            echo "latest=$(echo $TAG | cut -d- -f2)" >> $GITHUB_OUTPUT
```

Comment:
Remove these three `latest` definitions, as I did above, since it isn't used below anyway.

Reply:
Done in 27cd205